examples/small-study/bin/bundle-shield-studies-addon-utils.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# fail on any error
+set -o errexit
+
+# always run from the repository root directory
+script_path=`dirname $0`
+cd "$script_path/../../../"
+
+# paths
+WEBEXTAPIS_PATH="webExtensionApis"
+ADDON_SRC_PATH="examples/small-study/src"
+
+# bundle the study web extension experiment
+mkdir -p $ADDON_SRC_PATH/privileged/study
+cp $WEBEXTAPIS_PATH/study/api.js $ADDON_SRC_PATH/privileged/study/api.js
+cp $WEBEXTAPIS_PATH/study/fakeApi.js $ADDON_SRC_PATH/privileged/study/fakeApi.js
```

Comment:
fakeApi is so that we can advance on *trying* the api withougt having to actuaally have it all be working.  This comment reflects old work, when the patch was  a mess.